src/main/java/com/gradle/Overrides.java

```diff
@@ -1,23 +1,27 @@
 package com.gradle;
 
-import com.gradle.enterprise.gradleplugin.GradleEnterpriseBuildCache;
-import com.gradle.enterprise.gradleplugin.GradleEnterpriseExtension;
-import com.gradle.scan.plugin.BuildScanExtension;
+import com.gradle.ccud.adapters.reflection.ProxyFactory;
+import com.gradle.ccud.adapters.enterprise.proxies.BuildScanExtensionProxy;
+import com.gradle.ccud.adapters.enterprise.proxies.GradleEnterpriseBuildCacheProxy;
+import com.gradle.ccud.adapters.enterprise.proxies.GradleEnterpriseExtensionProxy;
 import org.gradle.api.provider.ProviderFactory;
 import org.gradle.caching.configuration.BuildCacheConfiguration;
 import org.gradle.caching.http.HttpBuildCache;
 
 import java.time.Duration;
 import java.util.Optional;
 
+import static com.gradle.ccud.adapters.enterprise.proxies.GradleEnterpriseBuildCacheProxy.gradleEnterpriseBuildCacheClass;
+import static com.gradle.ccud.adapters.enterprise.proxies.GradleEnterpriseBuildCacheProxy.isGradleEnterpriseBuildCache;
+
 /**
- * Provide standardized Gradle Enterprise configuration. By applying the plugin, these settings will automatically be applied.
+ * Provide standardized Develocity configuration. By applying the plugin, these settings will automatically be applied.
  */
 final class Overrides {
 
-    // system properties to override Gradle Enterprise configuration
-    static final String GRADLE_ENTERPRISE_URL = "gradle.enterprise.url";
-    static final String GRADLE_ENTERPRISE_ALLOW_UNTRUSTED_SERVER = "gradle.enterprise.allowUntrustedServer";
+    // system properties to override Develocity configuration
+    static final String DEVELOCITY_URL = "develocity.url";
```

Comment:
I think we need to check the old `gradle.enterprise.url` as well. This is for backward compatibility with the BVS.
I think the same is true for `allowUntrustedServer` and other system properties.

Reply:
The build validation scripts do not rely on these overrides. They configure the Develocity Gradle plugin directly via an init script.

Reply:
I restored old properties, and we will fall back to those if DV properties are not set. See https://github.com/gradle/common-custom-user-data-gradle-plugin/pull/273/commits/fc08a7d1f36eae6bdc1c24378c3c25321a00d37a